data/remnant/remnant 2 side missions.txt

```diff
@@ -0,0 +1,52 @@
+	to offer
+		has "event: remnant: cognizance calm"
+		has "Remnant: Broken Jump Drive 5: done"
+	on offer
+		event "remnant: emp ii deployment"
+		conversation
+			`You note a disturbance in the shipyards before the <ship> even touches the ground. Every pad is filled with ships, camels pulling wagon trains loaded with EMPs trundle out of the outfitter heading for ships, and technicians clamber everywhere. As you step out of your ship, Taely slides down the side of a Merganser resting next to you.`
+			`	"Good timing, Captain! We just rolled out an upgrade to some of our oldest equipment, and our Mergansers finally have something to carry as first-strike weapon. Interested in taking a look?"`
+			choice
+				`	"Yes"`
+				`	"No"`
+					goto empno
+			`	She turns and heads towards a nearby lift where another crew of Remnant had just moved several EMP torpedo canisters into place. "Well, you have probably seen us use EMP torpedoes now and then, right? Well, as you probably noticed, they have a few tactical issues. They are slow, they often hit our ships almost as much as our targets, and they take up space we could otherwise be using for more important things like laboratories or scanners." She taps a few commands into the console, and the lift begins raising the canisters towards the waiting ship.`
+			`	"Over the past decades we have had a few teams working on improving them from different angles. Some focusing on the torpedoes themselves, others focusing on the launcher. The surprising development, however, was from a team working down in the Warren. They figured out a way to shave a few cubic meters off the systems that assemble the torpedoes, and compress the whole works. Very impressive work, and the results should be in the outfitter for you in the next day or two."`
+			choice
+				`	"Thanks for the explanation."`
+					goto empno
+				`	"What does 'Warren' mean?"`
+			`	"It is our term for the network of caves, tunnels, and spaces deep underground. All three of our main worlds have them in some form or another. We know a little about the original people who built them, but not nearly as much as we would like. They called themselves the Zitchal Maugdrakoi, and one of the most distinguishing features we have been able to learn about them is that they do not seem to attach much importance to directionality. For example, in a control room we found, there were instruments and controls on every available surface, with just a scattering of handholds scattered around to support the operator." She shrugs. "Plume or Dusk would know more than I about them. I mostly focus on understanding their equipment."`
+			label empno
+			`	"Well, I should get back to work, then. Next time you are in space, take a look at how our ships fare in battle. Hopefully you will see an improvement." She grabs a couple components from a nearby lift, and scrambles back up the side of the Merganser to continue working on an EMP bay.`
+				accept
+	npc
```

Comment:
they did not fire a single EMP torpedo, maybe make sure to spawn those vairants?

Reply:
I had, for some reason, totally missed putting EMP pods on that Merganser variant... Thanks for pointing it out!